File: www/speech.js
    var exec = require('cordova/exec');

    function Speech() {
    }

    Speech.prototype.speak = function (text, successCallback, errorCallback) {
        exec(successCallback, errorCallback, "ie.kevinboyle.cordova.speech.Speech", "speak", [text]);
    };

    module.exports = new Speech();

File: src/wp/Speech.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WPCordovaClassLib.Cordova.Commands;
using Windows.Phone.Speech.Synthesis;
using WPCordovaClassLib.Cordova.JSON;
using Windows.Phone.Speech.Recognition;
using WPCordovaClassLib.Cordova;

namespace ie.kevinboyle.cordova.speech
{
    internal class Speech : BaseCommand
    {
        public void speak(string options)
        {
            string[] args = JsonHelper.Deserialize<string[]>(options);

            new SpeechSynthesizer().SpeakTextAsync(args[0]);
        }
    }
}
